datascience-notebook/tests/controls/nano_spec.rb

```diff
@@ -0,0 +1,19 @@
+title 'nano is available and default editor'
+
+control 'nano available and default editor' do
+  impact 'low'
```

Comment:
Huh, TIL, impact levels can also be strings! I've always used numerical values.

Reply:
Love how I accidentally discovered this on day 0 😬